docs/usage.md

```diff
@@ -0,0 +1,83 @@
+# 安装使用hostnic-cni
+
+## 使用k3s搭建环境
+
+* 安装cni标准插件
+```
+cd /opt/cni/bin/
+wget https://github.com/containernetworking/plugins/releases/download/v0.8.7/cni-plugins-linux-amd64-v0.8.7.tgz
+tar xvzf cni-plugins-linux-amd64-v0.8.7.tgz
+```
+
+* 使用运行时docker
+```
+curl https://releases.rancher.com/install-docker/19.03.sh | sh
+curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC='--flannel-backend=none --no-flannel' sh -s - --docker
+```
+* 使用运行时containerd
+```
+curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC='--flannel-backend=none --no-flannel' sh -
+```
+至此一个没有cni插件的单节点k8s集群搭建完成
+
+## 安装hostnic
+
+* 安装qingcloud IAAS secret
+```
+apiVersion: v1
+stringData:
+  config.yaml: |-
+    qy_access_key_id: 'xxxxx'
+    qy_secret_access_key: 'xxxxx'
+    # your instance zone
+    zone: "ap2a"
+kind: Secret
+metadata:
+  name: qcsecret
+type: Opaque
+
+```
+* 安装hostnic
+```
+kubectl apply -f https://raw.githubusercontent.com/yunify/hostnic-cni/master/deploy/hostnic.yaml
+```
+
+## 使用host
+
+* 给节点加上vxnet注解
+  `kubectl  annotate nodes i-xrwbww35  "hostnic.network.kubesphere.io/vxnet"="vxnet-cfn58ev"`
```

Comment:
Is it possible to have a set of nodes or all nodes share a set of vxnet?

Reply:
All nodes share a set of vxnet without add any annotations.

Reply:
I've added this configuration before, but thought I'd remove it. A vxnet can create 254 NICs, each host can mount 60 NICs, and a vxnet can only be shared by four hosts, so it's easy to exceed the limit, so it's mandatory to use annotation selection here, so that users can also plan their use

Reply:
> Is it possible to have a set of nodes or all nodes share a set of vxnet?

Yes

Reply:
> All nodes share a set of vxnet without add any annotations.

I've added this configuration before, but thought I'd remove it. A vxnet can create 254 NICs, each host can mount 60 NICs, and a vxnet can only be shared by four hosts, so it's easy to exceed the limit, so it's mandatory to use annotation selection here, so that users can also plan their use